package.json

```diff
@@ -20,15 +20,21 @@
     "url": "git://github.com/edwellbrook/node-tvdb.git"
   },
   "dependencies": {
-    "jszip": "3.1.2",
-    "request": "2.74.0",
-    "xml2js": "0.4.17"
+    "debug": "^2.2.0",
+    "lodash": "^4.15.0",
```

Comment:
Removed lodash usage where possible (tests).
index.js uses `defaults` and `flatten`. Changed these into `require('lodash/defaults')`. I prefer not to reinvent the wheel here and stick to using lodash, but including as little as needed.